tools/hangar/__snapshots__/test_corpus/valid/bring_local_dir.test.w_compile_tf-aws.md

```diff
@@ -249,6 +250,17 @@ module.exports = function({ $stdlib }) {
 //# sourceMappingURL=preflight.subdir2-5.js.map
 ```
 
+## preflight.subdir3-6.js
```

Comment:
this file feels like it shouldn't exist because there are no exports here (because there are no .w files in this directory).